.yarnrc.yml

```diff
@@ -132,48 +128,9 @@ packageExtensions:
   "@types/wordpress__plugins@3.0.0":
     peerDependencies:
       react: "*"
-  "@wordpress/block-library@6.0.18":
```

Comment:
Nice! I remember looking forward to this update knowing we could get rid of a ton of these extra WordPress rules for older packages. A while ago, I updated gutenberg to correctly specify peer dependencies, and the main consumer of the older versions was newspack-blocks -- until now!